### PR TITLE
Nt/tweaks

### PR DESCRIPTION
**Pull request by noumantahir**
### What does this PR?
checks if posts provided for parsing are comments.. avoid removing body for such posts.

### Where should the reviewer start?
dhive.js get posts methods
postParser.js `parsePosts` and `parsePost`

### Screenshots/Video
Screenshot 2021-03-23 at 11 43 59 AM

### CODE REVIEW

File: src/providers/hive/dhive.js [hunk 1]
```diff
@@ -484,9 +483,8 @@ export const getAccountPosts = async (query, currentUserName, filterNsfw) => {
     let posts = await client.call('bridge', 'get_account_posts', query);
 
     if (posts) {
-      posts = parsePosts(posts, currentUserName);
-
-      // FastImage.preload(posts.map(post=>({uri:post.image})))
+      const areComments = query.sort === 'comments';
```
Comment (feruzm) [inline src/providers/hive/dhive.js:486]: `|| query.sort === 'replies'`

Comment (feruzm) [reply src/providers/hive/dhive.js:486]: This is when viewing own profile.

Comment (noumantahir) [reply src/providers/hive/dhive.js:486]: checking

Comment (feruzm) [reply src/providers/hive/dhive.js]: @noumantahir pushed additions

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**